Tests/WorkspaceTests/ManifestSourceGenerationTests.swift

```diff
@@ -159,6 +159,35 @@ class ManifestSourceGenerationTests: XCTestCase {
         try testManifestWritingRoundTrip(manifestContents: manifestContents, toolsVersion: .v5_3)
     }
 
+    func testPackageDependencyVariations() throws {
+        let manifestContents = """
+            // swift-tools-version:5.3
```

Comment:
since there is logic that is tied to tools version in https://github.com/apple/swift-package-manager/pull/3415/files#diff-1f5839d3686c85003a930e9710b153b7897daa533cffb9463f81b1638510113aR30, should there be a test for > 5.4 in case there isn't already?

Reply:
That's a good point, and in fact the new test is tied to 5.5.

Reply:
The Plugin test is 5.5, but it makes sense to change this one to be 5.4 also.

Reply:
I have modified this test to be 5.4 and extended the checking to check for the expected presence of space depending on the tools version.